controllers: fix Numark DJ2GO2 Touch sliders and knobs

fjordstrom opened the pull request:
Fix input for sliders and knobs to use single byte value, since all
are on single CC status message.

Fix direction of tempo sliders.

Resolves: [lp1948596](https://bugs.launchpad.net/mixxx/+bug/1948596)

fjordstrom committed:
controllers: fix Numark DJ2GO2 Touch sliders and knobs

Fix input for sliders and knobs to use single byte value, since all
are on single CC status message.

Fix direction of tempo sliders.

Resolves: lp1948596

Swiftb0y commented on res/controllers/Numark_DJ2GO2_Touch_scripts.js:
```diff
@@ -91,7 +91,7 @@ DJ2GO2Touch.Deck = function(deckNumbers, midiChannel) {
     });
 
-    engine.setValue(this.currentDeck, "rate_dir", -1);
+    engine.setValue(this.currentDeck, "rate_dir", 1);
```
I don't think this is correct. This is actually user-configurable and should not be touched by the hardware mapping. What if you instead specify `invert: true` on the `tempoFader`? Thats what I had to do on my Numark Controller.

Swiftb0y reviewed (changes requested):
Hey there, thank you for taking the time to address a bug within mixxx.

fjordstrom committed:
controllers: remove rate_dir hardcoding from Numark DJ2GO2 Touch

remove line hardcoding tempo slider direction that affects user
configurable settings

Swiftb0y commented:
Great. So to confirm, irregardless of the actual slider direction in the mixxx UI, when you move the slider to the + region on the controller, the slider within also goes towards + (and vice versa)?

Swiftb0y commented:
In order to merge your PR, we must kindly ask you to sign our contributor agreement. Thanks.
https://docs.google.com/a/mixxx.org/spreadsheet/viewform?formkey=dEpYN2NkVEFnWWQzbkFfM0ZYYUZ5X2c6MQ 

Swiftb0y reviewed (approved):
LGTM. Thank you very much.

Swiftb0y commented:
Also quick question @fjordstrom:
In the bug report on launchpad, the reporter mentioned that their tempo faders don't center on 0%. This is somewhat common on individual units of cheap hardware. Does the incorrect center issue occur on your hardware as well?

Swiftb0y commented:
Ah, yes. Upon closer inspection of the XML I actually see the problem. My fault this slipped through the initial review. This makes sense. Thank you for clarifying and submitting this patch.
